mix.exs

```diff
@@ -57,7 +57,8 @@ defmodule Screenplay.MixProject do
       {:lcov_ex, "~> 0.2", only: [:dev, :test], runtime: false},
       {:sobelow, "~> 0.8", only: :dev},
       {:sentry, "~> 8.0"},
-      {:stream_data, "~> 0.5", only: :test}
+      {:stream_data, "~> 0.5", only: :test},
+      {:quantum, "~> 3.0"}
```

Comment:
It currently has a single instance. And honestly, I think it will only have a single instance for a long time. The userbase for Screenplay is very small so traffic should always be light enough so ECS does not scale us to use another task. In the next feature for Screenplay, we will likely be using a psql DB. When that happens, we could consider switching to [Oban](https://github.com/sorentwo/oban) for job processing. It has a feature that prevents duplicate jobs from running.